packages/mobile/src/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't think this should be added? As the celo-monorepo root yarn.lock is what's used